File: app/src/main/java/br/com/jonathanarodr/playmovie/api/ApiService.java
package br.com.jonathanarodr.playmovie.api;

import br.com.jonathanarodr.playmovie.BuildConfig;
import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public abstract class ApiService {

    public static Retrofit getInstance() {
        return new Retrofit.Builder()
                .baseUrl(BuildConfig.SERVER_URL)
                .addConverterFactory(GsonConverterFactory.create())
                .build();
    }

}

File: app/src/main/java/br/com/jonathanarodr/playmovie/api/ApiService.kt
package br.com.jonathanarodr.playmovie.api

import br.com.jonathanarodr.playmovie.BuildConfig
import retrofit2.Retrofit
import retrofit2.Retrofit.Builder
import retrofit2.converter.gson.GsonConverterFactory

object ApiService {
    @JvmStatic
    val instance: Retrofit get() = Builder()
        .baseUrl(BuildConfig.SERVER_URL)
        .addConverterFactory(GsonConverterFactory.create())
        .build()
}